spec/support/shared_metadata_examples.rb

```diff
@@ -0,0 +1,12 @@
+shared_examples 'a work with Tufts metadata attributes' do
```

Comment:
This is good for now. We should remove the boilerplate with a custom matcher before too long.